File: JAVA程序/Coll_Api/src/cn/itcast/generic/demo/Student.java
package cn.itcast.generic.demo;

import cn.itcast.p.bean.Person;

public class Student extends Person {

	public Student(String name, int age) {
		super(name, age);
		// TODO 自动生成的构造函数存根
	}

	public Student() {
		super();
		// TODO 自动生成的构造函数存根
	}
	@Override
	public String toString() {
		// TODO 自动生成的方法存根
		return this.getName()+"..学生.."+this.getAge();
	}
}

File: C++程序/简单学习/Student.cpp
#include "Student.h"
#include <iostream>
using std::cout;
using std::endl;
//Student类的方法实现
int Student::GetNo(){return No;}
int Student::GetAge(){return Age;}

Student::Student(int no,int age,char sex):No(no),Age(age),Sex(sex){}

void Student::ShowS(){
	cout<<"No="<<No<<'\t'<<"Age="<<Age<<'\t'<<"Sex="<<Sex<<endl;
}